### PR TITLE
feat(samples): add remaining featurestore api samples

### PR DESCRIPTION
**Pull request by suryaguthikonda**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-ai-platform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #313

### CODE REVIEW

**Review by diemtvu — APPROVED** [review]

File: samples/get-featurestore-sample.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ */
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/get-featurestore-sample.js:30]: Can you change this to 5s. For Get/List, the default timeout is 5s

File: samples/list-featurestores-async-sample.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * the code snippet
+ */
+
+'use strict';
+
+async function main(
+  project,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-featurestores-async-sample.js:29]: Same here, change to 5s

File: samples/list-featurestores-sample.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ * the code snippet
+ */
+
+'use strict';
+
+async function main(
+  project,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-featurestores-sample.js:29]: Same

File: samples/list-featurestores-stream-sample.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+ * the code snippet
+ */
+
+'use strict';
+
+async function main(
+  project,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-featurestores-stream-sample.js:29]: 5s

File: samples/update-featurestore-fixed-nodes-sample.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  fixedNodeCount = 1,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/update-featurestore-fixed-nodes-sample.js:31]: Can you change this to 10min

File: samples/update-featurestore-sample.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  minNodeCount = 1,
+  maxNodeCount = 3,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/update-featurestore-sample.js:32]: 10 min

File: samples/get-featurestore-sample.js (hunk 1 above)
Comment (suryaguthikonda) [reply samples/get-featurestore-sample.js]: Made the respective changes.

**Review by dizcology — DISMISSED** [review]

**Review by dizcology — APPROVED** [review]

**Pull request merged**